components/wallet-logger.js

```diff
@@ -27,8 +30,15 @@ export function WalletLogs ({ wallet, embedded }) {
         >clear logs
         </span>
       </div>
-      <div className={`${styles.logTable} ${embedded ? styles.embedded : ''}`}>
+      <div className={`${styles.tableContainer} ${embedded ? styles.embedded : ''}`}>
         <table>
+          <colgroup>
+            <col span='1' style={{ width: '1rem' }} />
+            <col span='1' style={{ width: '1rem' }} />
+            <col span='1' style={{ width: '1rem' }} />
+            <col span='1' style={{ width: '100%' }} />
+            <col span='1' style={{ width: '1rem' }} />
```

Comment:
This is kind of a hack to use as much space as possible for the actual log message and push `+` to the end. Instead of `width: '1rem'` I could have even used `width: 0px` since table columns seem to reserve width for the space they need independent of what I set here :thinking: 

Maybe I should have used a CSS grid when I created the wallet logs but table also works now ...

That reminds me I considered to add alternating colors for the rows but that messed with the context since the context rows should stay the same color.